src/frontends/pytorch/src/frontend.cpp

```diff
@@ -263,9 +263,8 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
         manager.register_pass<ov::frontend::pytorch::pass::GPTQMultPatternReplacer>();
     }
 
-    // the following 2 transformations are needed for keypoint detectron2 models to work.
+    // the following transformation is needed for keypoint detectron2 models to work.
     // AtenIndexToSelect will be called twice
-    manager.register_pass<ov::pass::ConvertConvertLike>();
     manager.register_pass<ov::frontend::pytorch::pass::AtenIndexToSelect>();
 
```

Comment:
@mvafin , do you know if the place for ConvertConvertLike had been chosen intentionally here and we can't move it from here? Or is it safe to move it to the end of the function? If latter, I won't remove it from here, but just add a second one in the end of the function.

Reply:
as I understand it, AtenIndexToSelect won't work after your change

Reply:
I am not sure about that. We could move it to the end, but we need to run https://github.com/openvinotoolkit/openvino/tree/master/tests/model_hub_tests/pytorch tests with -m nightly flag manually, since there is no CI worker for it that can be triggered for PR.